src/types.js

```diff
@@ -1,15 +1,48 @@
 // @flow
 
+import moment from 'moment'
+
 export type Uri = string
```

Comment:
Try using the new standard [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) type instead